.circleci/config.yml

```diff
@@ -0,0 +1,154 @@
+    command: bundle check || bundle install
+
+  - &save_cache
+    name: Save Bundler cache
+    key: rails-cache-<< parameters.version >>-<< parameters.rails-version >>-<< parameters.sequel-version >>-{{ checksum "Gemfile" }}
+    paths:
+      - vendor/bundle
+
+  - &run_tests
+     name: Run unit and integration tests
+     command: |
+       bundle exec rake
+
+references:
+  default_docker_ruby_executor: &default_docker_ruby_executor
+    image: circleci/ruby:<< parameters.version >>
+    environment:
+      BUNDLE_JOBS: 3
+      BUNDLE_RETRY: 3
+      BUNDLE_PATH: vendor/bundle
+      RAILS_VERSION: << parameters.rails-version >>
+      SEQUEL_VERSION: << parameters.sequel-version >>
+
+version: 2.1
+
+jobs:
+  tests_rails_3:
+    description: Build, unit and integration tests for Ruby
+    parameters:
+      version:
+        type: string
+      rails-version:
+        type: string
+      sequel-version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *credentials
+      - run: *check_bundler
+      - restore_cache: *restore_cache
+      - run: *install_bundler
+      - save_cache: *save_cache
+      - run: *run_tests
+  tests_rails_4:
+    description: Build, unit and integration tests for Ruby
+    parameters:
+      version:
+        type: string
+      rails-version:
+        type: string
+      sequel-version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *credentials
+      - run: *check_bundler
+      - restore_cache: *restore_cache
+      - run: *install_bundler
+      - save_cache: *save_cache
+      - run: *run_tests
+  tests_rails_5:
+    description: Build, unit and integration tests for Ruby
+    parameters:
+      version:
+        type: string
+      rails-version:
+        type: string
+      sequel-version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *credentials
+      - run: *check_bundler
+      - restore_cache: *restore_cache
+      - run: *install_bundler
+      - save_cache: *save_cache
+      - run: *run_tests
+  tests_rails_6:
+    description: Build, unit and integration tests for Ruby
+    parameters:
+      version:
+        type: string
+      rails-version:
+        type: string
+      sequel-version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *credentials
+      - run: *check_bundler
+      - restore_cache: *restore_cache
+      - run: *install_bundler
+      - save_cache: *save_cache
+      - run: *run_tests
+
+workflows:
+  version: 2
+  ci:
+    jobs:
+      - tests_rails_4:
+          matrix:
+            parameters:
+              version: ['2.2', '2.3', '2.4']
+              rails-version: ['4.2']
+              sequel-version: ['4.0']
+          filters:
+            tags:
+              only: /.*/
```

Comment:
What does this do, exactly?

Reply:
it should be deleted, it triggers your job on specific tags